ui/app/templates/storage/volumes/dynamic-host-volume.hbs

```diff
@@ -0,0 +1,115 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#each this.breadcrumbs as |crumb|}}
+  <Breadcrumb @crumb={{crumb}} />
+{{/each}}
+{{page-title "Dynamic Host Volume " this.model.name}}
+<section class="section with-headspace">
+  <h1 class="title" data-test-title>{{this.model.name}}</h1>
```

Comment:
We're not showing the ID anywhere on this page except the URL. That's probably ok? But maybe not what you intended.